src/python/visclaw/plotclaw.py

```diff
@@ -47,7 +53,90 @@ def plotclaw(outdir='.', plotdir='_plots', setplot = 'setplot.py',
     plotdata.format = format
     plotdata.msgfile = msgfile
 
-    plotpages.plotclaw_driver(plotdata, verbose=False, format=format)
+
+    frametools.call_setplot(plotdata.setplot, plotdata)
+
+
+    if plotdata.parallel:
+
+        # If this is the original call then we need to split up the work and 
+        # call this function again
+
+        # First set up plotdir:
+
+        plotdata._parallel_todo = 'initialize'
+        plotpages.plotclaw_driver(plotdata, verbose=False, format=format)
+
+        if frames is None:
+            if plotdata.num_procs is None:
+                plotdata.num_procs = int(os.environ.get("OMP_NUM_THREADS", 1))
+
+
+            frames = [[] for n in xrange(plotdata.num_procs)]
+            framenos = frametools.only_most_recent(plotdata.print_framenos,
+                                                   outdir)
+
+            # don't use more procs than frames or infinite loop!!
+            num_procs = min(plotdata.num_procs, len(framenos))
+
+            for (n, frame) in enumerate(framenos):
+                frames[n%num_procs].append(frame)
+
+            # Create subprocesses to work on each
+            plotclaw_cmd = "python $CLAW/visclaw/src/python/visclaw/plotclaw.py"
```

Comment:
This requires the CLAW environment variable to be set.  Couldn't we instead get the path using the location of the clawpack module?